src/components/App.tsx

```diff
@@ -226,7 +227,7 @@ export type PointerDownState = Readonly<{
   lastCoords: { x: number; y: number };
   // map of original elements data
   // (for now only a subset of props for perf reasons)
-  originalElements: Map<string, Pick<ExcalidrawElement, "x" | "y" | "angle">>;
+  originalElements: Map<string, NonDeleted<ExcalidrawElement>>;
```

Comment:
The comment above no longer makes sense. Are we ok with this change?

Reply:
Wanted to avoid this but from quick testing it seems cloning even ~1500 elements is fine (~2ms once at pointerdown), so I think we're good (not sure what my initial impetus was about).

@Jnforja let's remove the parenthesized comment above as @xixixao suggested, though.

Reply:
Done. Thanks for pointing that out. I didn't even notice it.